test/TorchSharpTest/TestTorchTensor.cs

```diff
@@ -7804,6 +7810,17 @@ public void TestHistogram()
             Assert.True(bin_edges.allclose(torch.tensor(new double[] { 0, 1, 2, 3 }), 0.001));
         }
 
+        [Fact]
+        [TestOf(nameof(torch.fake_quantize_per_channel_affine))]
+        public void TestFakeQuantizePerChannelAffine()
+        {
+            var x = torch.rand(2, 2, 2);
+            var scales = (torch.randn(2) + 1d) * 0.05d;
+            var zero_points = torch.zeros(2).to(torch.int32);
+            var result = torch.fake_quantize_per_channel_affine(x, scales, zero_points, axis: 0, quant_min: 0, quant_max: 255);
+            Assert.True(true);
```

Comment:
Huh? Is this just because the test is really just looking for an exception in the call?

Reply:
Took this code from the official documentation. It's the minimal amount of testing. 

Should test the result values, but would need to get rid of the random initializer.

Reply:
Either use a Generator instance, or (I do this frequently) just create a tensor from the random values in the PyTorch doc and then the correct results should be right there in the documentation already...